<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>722</width>
    <height>833</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <widget class="QWidget" name="widget" native="true">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QPushButton" name="btn_management">
             <property name="text">
              <string>Management</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_choose">
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="text">
              <string>Choose</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QLineEdit" name="line_choose">
         <property name="font">
          <font>
           <strikeout>false</strikeout>
          </font>
         </property>
         <property name="tabletTracking">
          <bool>false</bool>
         </property>
         <property name="focusPolicy">
          <enum>Qt::StrongFocus</enum>
         </property>
         <property name="contextMenuPolicy">
          <enum>Qt::DefaultContextMenu</enum>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="inputMask">
          <string/>
         </property>
         <property name="frame">
          <bool>true</bool>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label_hash_sha1">
           <property name="text">
            <string>SHA1:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="text_hash_sha1">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="tabletTracking">
            <bool>false</bool>
           </property>
           <property name="focusPolicy">
            <enum>Qt::StrongFocus</enum>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_hash_sha2">
           <property name="text">
            <string>SHA2:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="text_hash_sha2">
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_hash_md5">
           <property name="text">
            <string>MD5:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="text_hash_md5">
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="3">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btn_save">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_clear">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_show_data">
        <property name="text">
         <string>Show Database</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_delete">
        <property name="text">
         <string>Delete</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_load">
        <property name="text">
         <string>Load</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_check_data">
        <property name="text">
         <string>Check by data</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_logout">
        <property name="text">
         <string>Logout</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="2">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_check">
       <attribute name="title">
        <string>Check</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label_file_check">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="line_file_check_1">
            <property name="font">
             <font>
              <strikeout>false</strikeout>
             </font>
            </property>
            <property name="tabletTracking">
             <bool>false</bool>
            </property>
            <property name="focusPolicy">
             <enum>Qt::StrongFocus</enum>
            </property>
            <property name="contextMenuPolicy">
             <enum>Qt::DefaultContextMenu</enum>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="inputMask">
             <string/>
            </property>
            <property name="frame">
             <bool>true</bool>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="1">
         <layout class="QVBoxLayout" name="verticalLayout_13">
          <item>
           <widget class="QPushButton" name="btn_file_check">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>File Check</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_check">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>Check</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0" colspan="2">
         <layout class="QVBoxLayout" name="verticalLayout_12">
          <item>
           <widget class="QLabel" name="label_check_sha1_1">
            <property name="text">
             <string>SHA1:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="text_check_sha1_1">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="tabletTracking">
             <bool>false</bool>
            </property>
            <property name="focusPolicy">
             <enum>Qt::StrongFocus</enum>
            </property>
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="frameShape">
             <enum>QFrame::Box</enum>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_check_sha2_1">
            <property name="text">
             <string>SHA2:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="text_check_sha2_1">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="tabletTracking">
             <bool>false</bool>
            </property>
            <property name="focusPolicy">
             <enum>Qt::StrongFocus</enum>
            </property>
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="frameShape">
             <enum>QFrame::Box</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Sunken</enum>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_check_md5_1">
            <property name="text">
             <string>MD5:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="text_check_md5_1">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="tabletTracking">
             <bool>false</bool>
            </property>
            <property name="focusPolicy">
             <enum>Qt::StrongFocus</enum>
            </property>
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="frameShape">
             <enum>QFrame::Box</enum>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_data">
       <attribute name="title">
        <string>Data</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QTableWidget" name="table_file_hash"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>722</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btn_clear</sender>
   <signal>clicked()</signal>
   <receiver>text_hash_sha1</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>204</x>
     <y>795</y>
    </hint>
    <hint type="destinationlabel">
     <x>340</x>
     <y>345</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_clear</sender>
   <signal>clicked()</signal>
   <receiver>text_hash_sha2</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>204</x>
     <y>795</y>
    </hint>
    <hint type="destinationlabel">
     <x>318</x>
     <y>393</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_clear</sender>
   <signal>clicked()</signal>
   <receiver>text_hash_md5</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>204</x>
     <y>795</y>
    </hint>
    <hint type="destinationlabel">
     <x>340</x>
     <y>747</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_clear</sender>
   <signal>clicked()</signal>
   <receiver>line_choose</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>204</x>
     <y>795</y>
    </hint>
    <hint type="destinationlabel">
     <x>341</x>
     <y>143</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_logout</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>652</x>
     <y>781</y>
    </hint>
    <hint type="destinationlabel">
     <x>376</x>
     <y>800</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_clear</sender>
   <signal>clicked(bool)</signal>
   <receiver>text_check_sha1_1</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>204</x>
     <y>795</y>
    </hint>
    <hint type="destinationlabel">
     <x>482</x>
     <y>110</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_clear</sender>
   <signal>clicked(bool)</signal>
   <receiver>line_file_check_1</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>204</x>
     <y>795</y>
    </hint>
    <hint type="destinationlabel">
     <x>482</x>
     <y>102</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_clear</sender>
   <signal>clicked()</signal>
   <receiver>text_check_sha2_1</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>204</x>
     <y>795</y>
    </hint>
    <hint type="destinationlabel">
     <x>482</x>
     <y>110</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_clear</sender>
   <signal>clicked()</signal>
   <receiver>label_file_check</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>204</x>
     <y>795</y>
    </hint>
    <hint type="destinationlabel">
     <x>482</x>
     <y>102</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_clear</sender>
   <signal>clicked()</signal>
   <receiver>text_check_md5_1</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>204</x>
     <y>795</y>
    </hint>
    <hint type="destinationlabel">
     <x>482</x>
     <y>110</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
